scala app service sample

denvermb opened the pull request:
<!-- Thanks for creating a Pull Request! Before you submit, please read the contributing guide at https://github.com/Azure/AppService/blob/master/CONTRIBUTING.md -->

## Summary

<!-- Provide a description of what your pull request changes. -->

denverbrittain committed:
scala app service sample

JasonFreeberg commented:
## [Preview link](https://antares-blog-staging-pr-313.azurewebsites.net) 
- Your changes have been deployed to the preview site. The preview site will update as you add more commits to this branch. 
- The preview site shows any future-dated articles. Don't worry, if you are publishing a future-dated article, it will not show on the production site until the file's specified date.
- The preview link is shareable, but will be deleted when the pull request is merged or closed.
> *This is an automated message.*

Commit:
Merge branch 'master' into scala-app-service

msangapu-msft committed:
Update 2022-02-23-WordPress on App Service Public Preview.md (#280)

* Update 2022-02-23-WordPress on App Service Public Preview.md

* Update 2022-02-23-WordPress on App Service Public Preview.md

* Update 2022-02-23-WordPress on App Service Public Preview.md

* Update 2022-02-23-WordPress on App Service Public Preview.md

* Update 2022-02-23-WordPress on App Service Public Preview.md

Co-authored-by: Jason Freeberg <jafreebe@microsoft.com>

dependabot[bot] committed:
Bump nokogiri from 1.13.4 to 1.13.6 (#308)

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.13.4 to 1.13.6.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.13.4...v1.13.6)

---
updated-dependencies:
- dependency-name: nokogiri
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

JasonFreeberg commented on _posts/2022-05-31-scala-on-app-service.md:
```diff
@@ -0,0 +1,131 @@
+excerpt: "Run a Scala App using Java SE on App Service"
+category: java
+---
+
```
Can you add an introduction here? Introduce concepts (Scala, Play framework) and then use the generic, _follow the tutorial below to deploy a Play framework Scala app onto Azure App Service._ 

Really helps with SEO ;)

JasonFreeberg commented on _posts/2022-05-31-scala-on-app-service.md:
```diff
@@ -0,0 +1,131 @@
+
+If you do not have the required versions, follow these links to obtain them:
+
+* [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```
Replace Oracle JDK link with MSFT JDK link:

```suggestion
* [Java SE](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11)
```

JasonFreeberg commented on _posts/2022-05-31-scala-on-app-service.md:
```diff
@@ -0,0 +1,131 @@
+
+If you do not have the required versions, follow these links to obtain them:
+
+* [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [sbt](http://www.scala-sbt.org/download.html)
+* [maven](https://maven.apache.org/install.html) (Optional)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) (Optional)
```
you can move these hyperlinks into the bullets on lines 16-19, cuts down on length

JasonFreeberg commented on _posts/2022-05-31-scala-on-app-service.md:
Add a "Step 3" to check the app in the browser again

Add a small paragraph that gives the high-level technicalities of the fat JAR. (You explained it in our 1:1 before, like how the "assembly" command includes the whole Scala runtime or something to that effect)

dependabot[bot] committed:
Bump tar from 4.4.1 to 4.4.19 (#297)

Bumps [tar](https://github.com/npm/node-tar) from 4.4.1 to 4.4.19.
- [Release notes](https://github.com/npm/node-tar/releases)
- [Changelog](https://github.com/npm/node-tar/blob/main/CHANGELOG.md)
- [Commits](https://github.com/npm/node-tar/compare/v4.4.1...v4.4.19)

---
updated-dependencies:
- dependency-name: tar
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dfuenzalida commented on _posts/2022-05-31-scala-on-app-service.md:
```diff
@@ -0,0 +1,131 @@
+
+* Java 11
+* sbt 1.3.4 or higher. Note: if you downloaded this project as a zip file from <https://developer.lightbend.com>, the file includes an sbt distribution for your convenience.
+* Maven (Optional)
```
If you are using Maven commands, I think you need to remove the Optional label for Maven. Same on line 14.

yutanglin16 committed:
Updating script due to commandlet deprecation (#314)

Info from customer incident that: 

It looks like SecretValueText was deprecated from the commandlet
https://stackoverflow.com/questions/65825151/get-azkeyvaultsecret-cant-read-secret-value-in-powershell 

Updating the script to accommodate the deprecation.

Commit:
Merge branch 'scala-app-service' of https://github.com/Azure/AppService into scala-app-service

Commit:
update requested changes

Commit:
Update 2022-05-31-scala-on-app-service.md

Commit:
Update 2022-05-31-scala-on-app-service.md

Commit:
update port for sbt run and link for .jar local testing

Commit:
Update 2022-05-31-scala-on-app-service.md

Commit:
Update 2022-05-31-scala-on-app-service.md
